webapp/src/shared/context/SearchContext.tsx

```diff
@@ -39,7 +41,7 @@ function queryReducerCallback(state: Query, action: QueryReducerAction) {
   if (type === 'SEARCH') {
     return { search: action.value, offset: 0 };
   } else if (type === 'LOAD_MORE') {
-    return { ...state, offset: state.offset + action.limit };
+    return { ...state, offset: state.offset ?? 0 + action.limit };
```

Comment:
```suggestion
    return { ...state, offset: (state.offset ?? 0) + action.limit };
```
suggestion: The pagination doesn't work. These changes could fix it.

Reply:
Should be fixed.